collector/lib/DuplexGRPC.h

```diff
@@ -751,41 +751,41 @@ class StdoutDuplexClientWriter : public IDuplexClientWriter<W> {
     return Result(Status::OK);
   }
 
-  bool Sleep(const gpr_timespec& deadline) {
+  bool Sleep(const gpr_timespec& deadline) override {
     return true;
   }
 
-  Result WritesDoneAsync() {
+  Result WritesDoneAsync() override {
```

Comment:
For my info where can I find the definition of `override`?

Reply:
`override` is a keyword, it's specifying `WritesDoneAsync` is a virtual method that is also present in one of the classes this one inherits from, in this case it is defined as a pure virtual method in `IDuplexClient`.
https://github.com/stackrox/collector/blob/55187025b877e44f8b36e5b58f45b7469006d02d/collector/lib/DuplexGRPC.h#L239

https://en.cppreference.com/w/cpp/language/override

Reply:
Thanks!